script/tool/lib/src/update_excerpts_command.dart

```diff
@@ -89,6 +89,9 @@ class UpdateExcerptsCommand extends PackageLoopingCommand {
         if (!await _injectSnippets(example, targetPackage: package)) {
           return PackageResult.fail(<String>['Unable to inject excerpts']);
         }
+        if (!await _injectSnippets(example, targetPackage: example)) {
+          return PackageResult.fail(<String>['Unable to inject excerpts']);
+        }
```

Comment:
This needs unit tests (that it updates the example readme, that it fails if doing so fails).